lesson_07/script.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+const loading = document.getElementById("js-loading");
+const gif = document.createElement("img");
+gif.src = "loading-circle.gif";
+loading.appendChild(gif);
```

Comment:
```suggestion
const addLoading = （）=> {
 const loading = document.getElementById("js-loading");
 const gif = document.createElement("img");
 gif.src = "loading-circle.gif";
 loading.appendChild(gif);
}
```
Let's make it functional and do this in the right place.